stdlib/logging/config.pyi

```diff
@@ -5,14 +5,37 @@ from configparser import RawConfigParser
 from threading import Thread
 from typing import IO, Any, Optional, Union
 
+if sys.version_info >= (3, 8):
+    from typing import Literal
+else:
+    from typing_extensions import Literal
+
 if sys.version_info >= (3, 7):
     _Path = AnyPath
 else:
     _Path = StrPath
 
+DEFAULT_LOGGING_CONFIG_PORT: int
+RESET_ERROR: int  # undocumented
+IDENTIFIER: Any  # undocumented
```

Comment:
```suggestion
IDENTIFIER: re.Pattern[str]  # undocumented
```

Should work as well.

Reply:
You can do `from typing import Pattern` on all Python versions. On 3.7+ (not 3.8+) `re.pyi` just makes it available there too.